extensions/slack/src/search.tsx

```diff
@@ -1,11 +1,46 @@
 // This filename should be named `switch-to-channel.tsx` or something similar
 // but it's kept as `search.tsx` as changing the command's name will cause users to lose their keywords and aliases
-import { ActionPanel, Action, Icon, List } from "@raycast/api";
-
+import { ActionPanel, Action, Icon, List, getPreferenceValues } from "@raycast/api";
 import { User, useChannels } from "./shared/client";
 import { withSlackClient } from "./shared/withSlackClient";
 import { useFrecencySorting } from "@raycast/utils";
 import { OpenChannelInSlack, OpenChatInSlack, useSlackApp } from "./shared/OpenInSlack";
+import { convertSlackEmojiToUnicode, getTimeLocale } from "./shared/utils";
+
+const { displayExtraMetadata } = getPreferenceValues<Preferences.Search>();
+
+function getCoworkerTime(coworkerTimeZone: string): string {
+  const time = new Date();
+  return new Intl.DateTimeFormat(getTimeLocale(), {
+    timeZone: coworkerTimeZone,
+    hour: "2-digit",
+    minute: "2-digit",
+  }).format(time);
+}
+
+function searchItemAccessories(
+  statusEmoji: string | undefined,
+  statusText: string | undefined,
+  statusExpiration: string | null,
+  timezone: string,
+) {
+  const searchMetadata: Array<{ icon: string | Icon; text: string; tooltip?: string | null | undefined }> = [
+    {
+      icon: convertSlackEmojiToUnicode(statusEmoji ?? ""),
```

Comment:
What about custom icons in Slack? I guess they won't be shown, right?

Reply:
Yeah unfortunately those still come through as a plain string to whatever is set in Slack. My team has one that I use all the time, `:thinking_rotate:`, which will just come in as `:thinking_rotate:`, like emojis used to. Maybe we can try to figure out how to get these to show up somehow? 
![:thinking_rotate:](https://emoji.slack-edge.com/T01JAPU340P/thinking_rotate/7a69dc59fdcf22fa.gif)

Reply:
I guess we'll need to fetch custom emojis over [here](https://api.slack.com/methods/emoji.list), and then find the corresponding image. You can use `useCachedPromise` or `useFetch` as well for that. I guess there'll be a small delay for the user's statuses but I don't think that's a huge problem.

Reply:
By the way, if you don't have time for that, we can already ship it as is an improve it later. That's more of a small improvement rather than a blocking issue.